src/timesync/timesyncd-manager.h

```diff
@@ -42,11 +42,13 @@ struct Manager {
         LIST_HEAD(ServerName, system_servers);
         LIST_HEAD(ServerName, link_servers);
         LIST_HEAD(ServerName, fallback_servers);
+        LIST_HEAD(ServerName, runtime_servers);
```

Comment:
doesn't matter, but swap with previous lines. the "fallback" is conceptually the lst, and should probably be here, too